### DISCUSSION TITLE
Not identifying if a cookie is blocked

### DISCUSSION THREAD

**Discussion by helenafranca** [GoogleChromeLabs/ps-analysis-tool#535] [Support Forum] [answered]
- Why does PSAT show the message "we could not identify whether this cookie was blocked or not. Please take a look at the network panel by copying the filter string from the cookie's right-click context menu."? 
- What should I look for in the network panel to "identify whether this cookie was blocked or not"?
- The above message appears for cookies with First Party scope. Shouldn't I assume that cookies with First Party Scope will not break?

Please check: https://www.easy-shopping.com.ar/

**Comment by amedina** [comment, marked as answer]
PSAT uses multiple sources of data to gather information about Cookies and blocked reasons. What you are observing regarding "we could not identify the reason for a cookie being blocked" is an issue we have identified and will be improved in an upcoming version. 

First party cookies can be blocked for multiple reasons that are not related to the deprecation of third-party cookies. A common reason is `DomainMismatch: This cookie was blocked because the request URL's domain did not exactly match the cookie's domain, nor was the request URL's domain a subdomain of the cookie's Domain attribute value."
